NOTICE

```diff
@@ -4,4 +4,16 @@ Copyright 2017-present The Chubao Authors.
 This product includes software devoloped at
 The Apache Software Foundation (http://www.apache.org/).
 
+ChubaoFS learned and used some code from the MinIO project to implement
+S3 compatible object storage interface signature algorithm V2, signature
```

Comment:
the signature algorithm V2, signature ... for S3 compatible object storage interfaces